src/components/sections/people/PersonViewsPane.jsx

```diff
@@ -174,6 +174,8 @@ export default class PersonViewsPane extends RootPaneBase {
                             content={ viewItem.data.title }
                             onChange={ this.onChange.bind(this, 'title') }
                             placeholder={ this.props.intl.formatMessage({ id: 'panes.personViews.placeholders.title' }) }
+                            preventEnter={ true }
```

Comment:
I think this property should be renamed to `multiline` or something like that (which is the inverse of what it does now, so logic will need to be slightly altered as well).

Reply:
Right, so basically the prop will be multiline={false}?